community-operators/postgres-operator/postgres-operator.v3.5.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,352 @@
+            - "*"
+          - apiGroups:
+            - "*"
+            resources:
+            - nodes
+            verbs:
+            - get
+            - list
+      permissions:
+        - serviceAccountName: postgres-operator
+          rules:
+          - apiGroups:
+            - "*"
+            resources:
+            - "*"
+            verbs:
+            - "*"
+          - apiGroups:
+            - ""
+            resources:
+            - pods
+            - services
+            - endpoints
+            - persistentvolumeclaims
+            - events
+            - configmaps
+            - secrets
+            verbs:
+            - "*"
+          - apiGroups:
+            - apps
+            resources:
+            - deployments
+            - daemonsets
+            - replicasets
+            - statefulsets
+            verbs:
+            - "*"
+        - serviceAccountName: pgo-backrest
+          rules:
+          - apiGroups:
+            - ""
+            resources:
+            - pods
+            verbs:
+            - get
+            - list
+          - apiGroups:
+            - ""
+            resources:
+            - pods/exec
+            verbs: 
+            - create
+      deployments:
+        - name: postgres-operator
+          spec:
+            replicas: 1
+            selector:
+              matchLabels:
+                name: postgres-operator
+            strategy:
+              type: Recreate
+            template:
+              metadata:
+                labels:
+                  name: postgres-operator
+              spec:
+                serviceAccountName: postgres-operator
+                containers:
+                - name: apiserver
+                  image: crunchydata/pgo-apiserver:centos7-3.5.0
+                  imagePullPolicy: IfNotPresent
+                  ports:
+                  - containerPort: 8443
+                  env:
+                  - name: CRUNCHY_DEBUG
+                    value: "true"
+                  - name: PORT
+                    value: "8443"
+                  - name: NAMESPACE
+                    valueFrom:
+                      fieldRef:
+                        fieldPath: metadata.namespace
+                  - name: TLS_NO_VERIFY
+                    value: "false"
+                  volumeMounts:
+                    - mountPath: /pgo-config
+                      name: pgo-config
+                      readonly: true
+                    - mountPath: /pgo-auth-secret
+                      name: pgo-auth-secret
+                      readonly: true
+                - name: operator
+                  image: crunchydata/postgres-operator:centos7-3.5.0
+                  imagePullPolicy: IfNotPresent
+                  env:
+                  - name: CRUNCHY_DEBUG
+                    value: "true"
+                  - name: NAMESPACE
+                    valueFrom:
+                      fieldRef:
+                        fieldPath: metadata.namespace
```

Comment:
If this represents the operator's watch namespaces, this should be set to `metadata.annotations['olm.targetNamespaces']`

Reply:
This would apply to each of the containers

Reply:
I updated the fieldPath to use the annotations value for all containers.